src/core/mount.c

```diff
@@ -1168,6 +1169,16 @@ static void mount_enter_mounting(Mount *m) {
         m->control_command_id = MOUNT_EXEC_MOUNT;
         m->control_command = m->exec_command + MOUNT_EXEC_MOUNT;
 
+        dp = opendir(m->where);
```

Comment:
I don't think we want to nuke any files from the filesystem which are not under our direct control.